routes/theme_routes.py

```
@@ -0,0 +1,188 @@
+            return (
+                jsonify({"success": False, "message": f"主题不存在: {name}"}),
+                400,
+            )
+
+        manager = registry.hugo_manager
+
+        # 停止当前运行的服务器
+        if manager.is_running:
+            manager.stop()
+
+        # 临时使用环境变量覆盖持久化主题，启动服务器
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)